js/annotationtoolbar.mjs

```diff
@@ -98,41 +106,58 @@ export class AnnotationToolbar{
 
     }
     
-    setMode(){
-        let self=this;
+    /**
+     * Sets the current mode of the Annotation Toolbar based on the selection and active tool.
```

Comment:
This comment is not accurate. This should read "Sets the mode of the toolbar based on the currently selected items in the project. Individual tools will be enabled and disabled by this. If the currently active tool is not supported for the selected item(s) it will be deactivated." (The active tool is not used in making the decision of which tools should be enabled or disabled.)